src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -1521,6 +1521,28 @@ private bool CheckIsVariable(SyntaxNode node, BoundExpression expr, BindValueKin
                 return true;
             }
 
+            var conditional = expr as BoundConditionalOperator;
+            if (conditional != null)
```

Comment:
I have another change where this whole sequence of ifs is refactored into one big switch.

---
In reply to: [105260877](https://github.com/dotnet/roslyn/pull/17436#discussion_r105260877) [](ancestors = 105260877)